Sources/SwifterSwift/UIKit/UIControlExtensions.swift

```diff
@@ -0,0 +1,31 @@
+// UIControlExtensions.swift - Copyright 2022 SwifterSwift
+
+#if canImport(UIKit)
+import UIKit
+
+// MARK: - Methods
+
+public extension UIControl {
+    
+    /// SwifterSwift: Perform action on specified event.
+    ///
+    /// - Parameters:
+    ///   - event: Event to perform action for.
+    ///   - action: Action to perform.
+    func on(_ event: UIControl.Event, perform action: @escaping () -> Void) {
```

Comment:
I chose `on(event:perform)` instead of something like `addAction`, cause I want to add similar extensions to `UIView` `onTap(perform:)`. It's also provide ability to extend it even further to something like `onTouchUpInside`.